packages/rspack/src/config/defaults.ts

```diff
@@ -307,6 +310,14 @@ const applyModuleDefaults = (
 	assertNotNill(module.parser.javascript);
 	applyJavascriptParserOptionsDefaults(module.parser.javascript);
 
+	F(module.parser, JSON_MODULE_TYPE, () => ({}));
+	assertNotNill(module.parser[JSON_MODULE_TYPE]);
+	D(
+		module.parser[JSON_MODULE_TYPE],
+		"exportsDepth",
+		mode === "development" ? 1 : Number.POSITIVE_INFINITY
```

Comment:
```suggestion
		mode === "development" ? 1 : Number.MAX_SAFE_INTEGER
```

Reply:
It is `Infinity` in webpack. Should we align?

Reply:
I think both is fine, but I'm not sure what will pass to rust side by napi-rs if we use Infinity here

Reply:
Make sense. After tested napi locally. I Found that:

js pass `Infinity` & rust declare `f64` =>  exportsDepth: f64::INFINITY(rust side)
js pass `Infinity` & rust declare `u32` =>  exportsDepth: 0

js pass `MAX_SAFE_INTEGER` & rust declare `f64` =>  exportsDepth: 9007199254740991 (2^53-1)
js pass `MAX_SAFE_INTEGER` & rust declare `u32` =>  exportsDepth: 4294967295 (2^32-1)

Should we keep `f64` + `Infinity` to align?

Reply:
Both fine to me, but I prefer `u32` a little bit :)

Reply:
> Both fine to me, but I prefer `u32` a little bit :)

No problem. Let we use `u32`. I think it's more efficient in rust side. 

When there is feedback from users, we can make further changes.

I will update it later.